pkg/bot/interactive/message.go

```diff
@@ -128,23 +119,23 @@ type Selects struct {
 	Items []Select
 }
 
-// Input is used to create input elements to use in slack messages.
-type Input struct {
-	ID               string
-	DispatchedAction bool
-	Element          InputElement
-	Label            InputLabel
-}
+// DispatchedInputAction defines when the action should be sent to our backend.
+type DispatchedInputAction string
 
-// InputElement is one of the components of Input. This component is mostly used to hold elements like input text, etc...
-type InputElement struct {
-	Type PlainTextInputType
-}
+// Defines the possible options to dispatch the input action.
+const (
+	NoDispatchInputAction          DispatchedInputAction = ""
+	DispatchInputActionOnEnter     DispatchedInputAction = "on_enter_pressed"
+	DispatchInputActionOnCharacter DispatchedInputAction = "on_character_entered"
+)
 
-// InputLabel refers to label of input element
-type InputLabel struct {
-	Type PlainTextType
-	Text string
+// PlaintextInput is used to create input elements to use in slack messages.
+type PlaintextInput struct {
```

Comment:
Would this make sense as a `LabelInput`? _A label will always be text based (similar to HTML label tags)_ 

This is just a general suggestion, not a blocker. But it could improve readability.

Reply:
Thanks, suggestion applied 👍